operational/azure/total_instance_hours/azure_usage_instance_hours_used.pt

```diff
@@ -0,0 +1,412 @@
+    })
+    count_1++
+    if (count_1 < _.size(group_by_inst_fam)) { chart_data = chart_data + "|" }
+  })
+
+  //Create Chart Title
+  var policy_title = "Total Instance Hours Used Per Instance Family For " + param_region
+  var chart_title = "chtt=" + policy_title
+  if ( param_region == "" ) {
+    policy_title = "Total Instance Hours Used Per Instance Family"
+    chart_title = "chtt=" + policy_title
+  }
+
+  //Whole Chart object
+  var chart = {
+    chart_type: encodeURI("cht=bvs"),
+    chart_size: encodeURI("chs=900x500"),
+    chart_data: encodeURI(chart_data),
+    chart_title: encodeURI(chart_title),
+    chart_image: encodeURI("chof=.png"),  
+    chart_label_position: encodeURI("chdlp=b"),
+    chart_axis: encodeURI("chxt=y,x"),
+    chart_axis_label: encodeURI(chart_axis_labels),
+    chart_axis_format: encodeURI("chxs=0N*f" + "0sz* Hours"),
+    chart_line_style: encodeURI("chls=3|3|3|3|3|3|3|3|3|3|3"),
+    chart_line_color: encodeURI("chco=6929c4,9f1853,198038,b28600,1192e8,009d9a,005d5d,007d79"),
+    chart_data_scale: encodeURI("chds=a"),
+    chart_legend: encodeURI(chart_legend),
+    chart_legend_size: encodeURI("chdls=000000,5"),
+    policy_title: policy_title
+  }
+
+  report[0]["chart_dimensions"] = chart
+  EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_inst_hrs_per_inst_fam" do
+  validate_each $ds_chart_creation do
+    summary_template "Azure Usage Report - Number of Instance Hours Used (Normalized - past 12 months)"
+    detail_template <<-EOS
+# Azure - {{ with index data 0 }}{{ .chart_dimensions.policy_title }}{{ end }} Report
+![Instance Hours Used Per Instance Family Chart](https://image-charts.com/chart?{{ with index data 0 }}{{ .chart_dimensions.chart_data }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_size }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_type }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_image }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_title }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_label_position }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_axis }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_axis_label }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_line_style }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_line_color }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_data_scale }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_legend_size }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_legend }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_axis_format }}{{ end }})
+___
+###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
+EOS
+    check eq(0,1)
```

Comment:
Line 385: Possible invalid spacing between comma-separated items found.
Comma separated items should be organized as follows, with a single space following each comma: apple, banana, pear

Reply:
Updated